app/src/main/java/sensors_in_paradise/sonar/SettingsActivity.kt

```diff
@@ -21,8 +27,53 @@ class SettingsActivity : AppCompatActivity() {
     }
 
     class SettingsFragment : PreferenceFragmentCompat() {
+        private var bgPrefs: SwitchPreference? = null
+
```

Comment:
Imo a more descriptive name would be better here. E.g. `editorStickmanBackgroundSwitchPreference` or just `editorStickmanBackgroundPref`